lib/base/networking/apis/auth_handler.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:cookie_jar/cookie_jar.dart';
+import 'package:dio/dio.dart';
+import 'package:dio_cookie_manager/dio_cookie_manager.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart' as webview;
+import 'package:gocast_mobile/base/helpers/model_generator.dart';
+import 'package:gocast_mobile/model/token_model.dart';
+import 'package:gocast_mobile/model/user_model.dart';
+import 'package:gocast_mobile/routes.dart';
+
+class AuthHandler {
+  /* static Future<void> saveToken(List<Cookie> cookies) async {
+    for (var cookie in cookies) {
+      if (cookie.name == 'jwt') {
+        SharedPreferences prefs = await SharedPreferences.getInstance();
+        await prefs.setString('jwt', cookie.value);
+
+        // DEBUG: Check cookie
+        String? jwt = prefs.getString('jwt');
+        debugPrint('Current jwt: $jwt');
+        return;
+      }
+    }
+
+    // TODO: Handle error when no jwt cookie is found
+    debugPrint('No JWT cookie found');
+  }
+*/
```

Comment:
It's always a good practice not to push commented-out code - [here](https://kentcdodds.com/blog/please-dont-commit-commented-out-code)

Reply:
My mistake, forgot to exclude it before committing :sweat_smile:  - should be fixed now.